config/default.yml

```diff
@@ -1667,6 +1667,11 @@ Lint/NextWithoutAccumulator:
   Enabled: true
   VersionAdded: '0.36'
 
+Lint/NoReturnInBeginEndBlocks:
+  Description: 'Do not `return` inside `begin..end` blocks in assignment contexts.'
+  Enabled: pending
+  VersionAdded: '0.93'
```

Comment:
Please change to '1.1' 😅

Reply:
Sure, no worries. I updated it to `1.1` and updated the doc as well. 👍

Reply:
Of course 1.1 has been released two days ago 🤣 

I'll merge and update myself...